control-plane/config/eventing-kafka-broker/100-source/100-kafka-source.yaml

```diff
@@ -40,6 +40,14 @@ spec:
           #
           # see issue: https://github.com/knative/serving/issues/912
           x-kubernetes-preserve-unknown-fields: true
+          properties:
+            spec:
```

Comment:
I think I'd like to see the other "spec" properties first, before we _just_ add this here

Reply:
That's refactoring of the crd. I prefer to change in other prs.

Reply:
yeah, exactly. That's why I said before to isolate the change, and land it _before_ just doing this one